packages/web/components/Setup/SetupSkills.tsx

```diff
@@ -44,6 +44,7 @@ export const SetupSkills: React.FC<SetupSkillsProps> = ({
     });
 
     if (error) {
+      console.warn(error); // eslint-disable-line no-console
```

Comment:
I dont think these are necessary. Really we should be using some logging tool like Sentry to post unexpected errors to a Discord channel... but that's for another ticket :)

Reply:
I'd be fascinated to use some sort of centralized logging system. Since Vercel and Github are getting Hasura timeouts, I assume whatever is generating the production site is getting them as well.